src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -55,8 +57,14 @@ def name(self):
     #################
 
     def setup(self, builder: Builder):
+        self.us_population_size = builder.configuration.us_population_size
         self.start_time = get_time_stamp(builder.configuration.time.start)
         self.randomness = builder.randomness.get_stream(self.name)
+        # create a randomness stream for business moves; use a common random number
+        # for identical business move behavior across parallel sims
+        self.business_moves_randomness = RandomnessStream(
+            key="business_moves", clock=builder.time.clock(), seed=12345
```

Comment:
you could use the draw here to allow different draws to have different business dynamics and still have them be common across seeds (did i get the terminology right?)

Reply:
I'm not exactly sure what you mean here. We are only running the simulation with a single draw. If we did have different draws they would result in different business dynamics if we had draw level parameterization of those dynamics (say if move rates were specified at the draw level).

Reply:
This brings up a good point - we've kind of set a requirement that we run with one draw; should we actually enforce that and throw a runtime error if otherwise?